gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -18012,13 +18067,18 @@ properties:
         - Ovary, NOS
         - Ovary, Right
         - Pancreas
+        - Parametrium, left
+        - Parametrium, right
         - Parotid gland
         - Pelvic Peritoneum
+        - Pelvic lymph node(s)
```

Comment:
This is "Pelvic Lymph Node(s)" in jira.

Reply:
Updated to what's in JIRA. It looks like past releases were not consistent with case of enums, will probably need to bring up in data dict meeting and discuss if a large set of migrations to be consistent will be needed...